website/content/api-docs/system/internal-counters.mdx

```diff
@@ -89,9 +89,13 @@ $ curl \
 
 ## Client Count
 
-This endpoint returns the number of clients per namespace, as the sum of active entities and non-entity tokens.
+This endpoint returns the number of clients per namespace, as the sum of clients calculated from the use 
+of active entities and non-entity tokens.
 An "active entity" is a distinct entity that has created one or more tokens in the given time period.
-A "non-entity token" is a token with no attached entity ID.
+A "non-entity token" is a token with no attached entity ID. 
+Both non-entity tokens and active entities have distinct client IDs. For more information on how clients
+these client IDs, and how clients are counted, please visit the 
```

Comment:
`how clients these client IDs` --> should this be something like `how clients map to these client IDs` or similar ?

Reply:
Yes, thank you!